pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -81,6 +81,7 @@ func GetBootstrapSecurityContextConstraints(sccNameToAdditionalGroups map[string
 			SupplementalGroups: kapi.SupplementalGroupsStrategyOptions{
 				Type: kapi.SupplementalGroupsStrategyRunAsAny,
 			},
+			SeccompProfiles: []string{"*"},
```

Comment:
oh, and this was changed specifically because the seccomp feature conformance tests need to be able to specify the annotation.